src/CodeStyle/Core/Analyzers/PublicAPI.Unshipped.txt

```diff
@@ -1,100 +1 @@
-Microsoft.CodeAnalysis.Internal.Editing.DeclarationKind
```

Comment:
This PR is targeting the 17.4 release branch. Don't we usually do this operation in the `main` branch?

Reply:
We will have to do in both branches. We'll port from here back to main

Reply:
@jcouv no, we usually do this in the release branch and let that flow to main.